x/stake/tick.go

```diff
@@ -6,8 +6,8 @@ import (
 )
 
 const (
-	hrsPerYr  = 8766 // as defined by a julian year of 365.25 days
-	precision = 1000000000
+	hrsPerYr  = 8766         // as defined by a julian year of 365.25 days
+	precision = 100000000000 // increased to this precision for accuracy with tests on tick_test.go
```

Comment:
I increased  the var precision in tick.go, because some of the tests  in tick_test.go it would randomly remove a small amount of bonded shares (like 13 out of 5000000) and it would read no change in inflation, because the rational number num and denom precision were cutting off too early, and considering inflation the same. Not sure if changing the var precision is o.k. though, maybe that precision was chosen for a reason

Reply:
very interesting thanks for this comment